### ISSUE TITLE
Directory Profiles "Expand Introduction" link does not meet minimum color contrast

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#8572]
Siteimprove is reporting that "Color contrast does not meet minimum requirement" for a link on profiles from Directory Profiles.

Examples:

**Tippie**

https://my2.siteimprove.com/Inspector/989984/A11Y/Page?pageId=31563701187&impmd=264ED37B65FA6186129334A508391970#/sia-r69/failed/cVZTMktpMVA=;ZTdmNDE0Mjc=

**Education**

https://my2.siteimprove.com/Inspector/988894/A11Y/Page?pageId=34679773117&impmd=264ED37B65FA6186129334A508391970#/sia-r69/failed/cVZTMktpMVA=;OWViY2JiYTc=

We need to increase the color contrast to 4.5:1 or higher.

# Question

Is this something we can tackle or does it need addressed on the Directory Profiles side of things?

**Comment by briand44** [comment]
Sounded like this should be addressed on the Profiles side of things. I emailed Denny and passed this along. Going to close as not planned.
